app/api/db/__init__.py

```diff
@@ -0,0 +1,213 @@
+import os
+import urllib.parse
+from contextlib import contextmanager
+from dataclasses import dataclass
+from typing import Any, Generator, Optional
+
+import psycopg2
+import sqlalchemy.pool as pool
+from sqlalchemy import create_engine
+from sqlalchemy.engine import Engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+
+import api.logging
+from api.db.migrations.run import have_all_migrations_run
+
+logger = api.logging.get_logger(__name__)
+
+
+@dataclass
+class DbConfig:
+    host: str
+    name: str
+    username: str
+    password: Optional[str]
+    schema: str
+    port: str
+    hide_sql_parameter_logs: bool = True
+
+
+def get_db_config() -> DbConfig:
+    db_config = DbConfig(
+        host=os.getenv("DB_HOST", "localhost"),
+        name=os.getenv("POSTGRES_DB", "mock-api"),
+        username=os.getenv("POSTGRES_USER", "mock_api_user"),
```

Comment:
What about dropping the "mock"?
```suggestion
        name=os.getenv("POSTGRES_DB", "api"),
        username=os.getenv("POSTGRES_USER", "api_user"),
```

Reply:
I want to be careful about making the names TOO simple like "api" or "db" as that can make things more difficult to debug when you can't figure out what parameter you need to change (speaking from experience while writing this).

I was envisioning the future when this is a template library and the "mock" bit here gets replaced with your project name (eg. `mass-pfml-api` or `nj-ui-api`). But I'm open to alternative ideas.

Reply:
We discussed this synchronously and decided on `main-api` and `main-db`.